packages/cli/src/commands/init/init.ts

```diff
@@ -175,13 +176,31 @@ async function installDependencies({
   loader.succeed();
 }
 
+function checkVersionAndTemplate(options: Options, version: string): string {
+  const isTypescriptTemplate = options.template?.includes(
+    'react-native-template-typescript',
+  );
+
+  if (isTypescriptTemplate && options.template?.includes('@')) {
+    const typescriptTemplateVersion = options.template.split('@')[1];
+    if (semver.gte(typescriptTemplateVersion, '0.71.0')) {
+      logger.warn(
+        "Starting from verision '0.71' typescript is used by default.\nDiregarding template and using default react-native",
```

Comment:
There's also a typo in "verision". My suggestion:
```suggestion
        "Ignoring custom template: 'react-native-template-typescript'. Starting from React Native v0.71 TypeScript is used by default.",
```